base/MAPL_SimpleBundleMod.F90

```diff
@@ -762,6 +762,7 @@ Function MAPL_SimpleBundleRead (filename, bundle_name, grid, time, verbose, &
     logical, OPTIONAL,           intent(in)    :: verbose
     character(len=*), optional,  intent(IN)    :: only_vars 
     character(len=*), optional,  intent(IN)    :: expid
+    logical,          optional,  intent(IN)    :: voting
```

Comment:
New optional arguments are now required to be after an "unusable" argument.  This enforces keyword use for such arguments and ensures backward compatibility as arguments are added.  (MAPL 3 will move the existing optionals to after this as well.)
```suggestion
    class(KeywordEnforcer), optional, intent(in) :: unused
    logical,          optional,  intent(in)    :: voting
```